modules/perception/camera/tools/offline/offline_obstacle_pipeline.cc

```diff
@@ -44,12 +62,28 @@ DEFINE_string(params_dir, "/apollo/modules/perception/data/params",
 DEFINE_string(visualize_dir, "/tmp/0000", "visualize dir");
 DEFINE_double(camera_fps, 15, "camera_fps");
 DEFINE_bool(do_undistortion, false, "do_undistortion");
-DEFINE_string(undistortion_save_dir, "", "save imgs dir after undistortion");
+DEFINE_string(undistortion_save_dir, "./undistortion_result",
+              "save imgs dir after undistortion");
```

Comment:
Comments and descriptions should be closer to general sentences:
"Save images dir after undistortion."

Please also fix others.